mintlify/snippets/platform-configuration.mdx

```diff
@@ -0,0 +1,142 @@
+      "currencyCode": "USD",
+      "minAmount": 100,
+      "maxAmount": 1000000,
+      "requiredCounterpartyFields": [
+        { "name": "FULL_NAME", "mandatory": true },
+        { "name": "BIRTH_DATE", "mandatory": true }
+      ],
+      "enabledTransactionTypes": ["OUTGOING", "INCOMING"],
+      "umaProviderRequiredCustomerFields": [
+        "NATIONALITY",
+        "BIRTH_DATE"
+      ],
+      "umaProviderRequiredCounterpartyCustomerFields": [
+        "FULL_NAME",
+        "COUNTRY_OF_RESIDENCE"
+      ]
+    }
+  ],
+  "isRegulatedFinancialInstitution": false,
+  "createdAt": "2023-06-15T12:30:45Z",
+  "updatedAt": "2023-07-01T10:00:00Z"
+}
+```
+</ResponseExample>
+
+<Note>
+If this is your first time configuring the platform, some default values may be returned which were set up when you first created your account.
+</Note>
+
+### Update Platform Configuration
+
+```http
+PATCH /config
+```
+
+<RequestExample>
+```json Request body
+{
+  "umaDomain": "mycompany.com",
+  "webhookEndpoint": "https://api.mycompany.com/webhooks/uma",
+  "supportedCurrencies": [
+    {
+      "currencyCode": "USD",
+      "minAmount": 100,
+      "maxAmount": 1000000,
+      "enabledTransactionTypes": ["OUTGOING", "INCOMING"],
+      "requiredCounterpartyFields": [
+        { "name": "FULL_NAME", "mandatory": true },
+        { "name": "BIRTH_DATE", "mandatory": true },
+        { "name": "ADDRESS", "mandatory": false }
+      ]
+    }
+  ]
+}
+```
+</RequestExample>
+
+<ResponseExample>
+```json Success
+{
+  "id": "PlatformConfig:019542f5-b3e7-1d02-0000-000000000003",
+  "umaDomain": "mycompany.com",
+  "proxyUmaSubdomain": "mycompany",
+  "webhookEndpoint": "https://api.mycompany.com/webhooks/uma",
+  "supportedCurrencies": [
+    {
+      "currencyCode": "USD",
+      "minAmount": 100,
+      "maxAmount": 1000000,
+      "requiredCounterpartyFields": [
+        { "name": "FULL_NAME", "mandatory": true },
+        { "name": "BIRTH_DATE", "mandatory": true },
+        { "name": "ADDRESS", "mandatory": false }
+      ],
+      "enabledTransactionTypes": ["OUTGOING", "INCOMING"],
+      "umaProviderRequiredCustomerFields": [
+        "NATIONALITY",
+        "BIRTH_DATE"
+      ],
+      "umaProviderRequiredCounterpartyCustomerFields": [
+        "FULL_NAME",
+        "COUNTRY_OF_RESIDENCE"
+      ]
+    }
+  ],
+  "isRegulatedFinancialInstitution": false,
+  "createdAt": "2023-06-15T12:30:45Z",
+  "updatedAt": "2023-06-15T12:30:45Z"
+}
+```
+</ResponseExample>
+
+<Tip>
+Fields such as `id`, `createdAt`, `updatedAt`, `isRegulatedFinancialInstitution`, and `proxyUmaSubdomain` are read-only and returned by GET responses; they cannot be modified via PATCH. The arrays `umaProviderRequiredCustomerFields` and `umaProviderRequiredCounterpartyCustomerFields` are also read-only and reflect provider-enforced requirements.
+</Tip>
+
+### Configuration Parameters
+
+- **umaDomain**: Domain part of UMA addresses for your users (e.g., `$username@mycompany.com`).
```

Comment:
I think we probably want to reorder these and give a bit more context because some of them are only really relevant for the UMAaaS use case. I think the ones that must be there is the webhookEndpoint and supported currencies, but the other stuff is all kind of optional depending on your use case. We should probably downplay some of the UMA config stuff in the shared snippet version of this.